### PR TITLE
Remove ' from --enable option value

### LINKED ISSUE #2938: Fix command to enable service catalog in experimental features doc
### General information
Command to enable service catalog in [Experimental Features](https://github.com/minishift/minishift/blob/master/docs/source/using/experimental-features.adoc#enabling-experimental-oc-cluster-up-flags) doc fails with error:
```
- Starting OpenShift cluster -- Extra 'oc' cluster up flags (experimental) ... 
   '--enable=*,service-catalog''
.Error during 'cluster up' execution: Error starting the cluster. ssh command error:
command : /var/lib/minishift/bin/oc cluster up --routing-suffix 192.168.42.250.nip.io --base-dir /var/lib/minishift/base --image 'openshift/origin-${component}:v3.11.0' --public-hostname 192.168.42.250 --enable=*,service-catalog'
err     : exit status 1
output  : bash: -c: line 0: unexpected EOF while looking for matching `''
bash: -c: line 1: syntax error: unexpected end of file
```

### PR DESCRIPTION
**Pull request by dgolovin**
Fix #2938.

With ' in option value minishift fails to start with error

```
Starting OpenShift cluster -- Extra 'oc' cluster up flags (experimental) ...
   '--enable=*,service-catalog''
.Error during 'cluster up' execution: Error starting the cluster. ssh command error:
command : /var/lib/minishift/bin/oc cluster up --routing-suffix 192.168.42.250.nip.io --base-dir /var/lib/minishift/base --image 'openshift/origin-${component}:v3.11.0' --public-hostname 192.168.42.250 --enable=*,service-catalog'
err     : exit status 1
output  : bash: -c: line 0: unexpected EOF while looking for matching `''
bash: -c: line 1: syntax error: unexpected end of file
```

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
ok to test

File: docs/source/using/experimental-features.adoc [hunk 1]
```diff
@@ -42,7 +42,7 @@ $ MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags "--ser
 For OKD 3.10.0 or later the command is:
 
 ----
-$ MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags "--enable=*,service-catalog'"
+$ MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags "--enable=*,service-catalog"
```
Comment (praveenkumar) [inline docs/source/using/experimental-features.adoc]: :+1:

**Comment by LalatenduMohanty** [comment]
> @dgolovin just before the line you have modified, there is an example for OKD 3.9, Can you please remove it as in Minishift 1.26.0 we do not support OKD 3.9.0.

If you are ok with it, I can do the required change while merging this PR. As the extra change, I suggested is a very small one I do not see the requirement for a new PR. So let me know what you prefer.

**Comment by dgolovin** [comment]
@LalatenduMohanty why? this fix is for specific issue, If there is another problem with doc, let's open issue for that and fix it with different patch.

**Comment by gbraad** [comment]
I think a new issue for the removal of the  non-supported version is needed. Let's just merge this...

**Pull request merged**